Northstar.CustomServers/mod/scripts/vscripts/mp/_gamestate_mp.nut

```diff
@@ -868,3 +878,126 @@ float function GetTimeLimit_ForGameMode()
 	// default to 10 mins, because that seems reasonable
 	return GetCurrentPlaylistVarFloat( playlistString, 10 )
 }
+
+// faction dialogue
+void function KilledPlayerTitanDialogue( entity attacker, entity victim )
+{
+	if( !attacker.IsPlayer() )
+		return
+	entity titan
+	if ( victim.IsTitan() )
+		titan = victim
+
+	if( !IsValid( titan ) )
+		return
+	string titanCharacterName = GetTitanCharacterName( titan )
+
+	switch( titanCharacterName )
```

Comment:
this feels weird, is there a better way to do this than a switch statement that wouldn't support additional/modded titans?

Reply:
modded titans and monarchs will use the default "kc_pilotkilltitan" I tested, should be alright

Reply:
We should probably avoid switch statements like this eventually, when we come up with better support for modded titans, there are a bunch of switch statements like this in various places, which should all be replaced with table indexing imo, so that new entries can be added at runtime